plugins/modules/device_credential_workflow_manager.py

```diff
@@ -1041,20 +1023,20 @@ def get_cli_params(self, cliDetails):
             in the format suitable for the Cisco Catalyst Center config.
         """
 
-        cliCredential = []
-        for item in cliDetails:
+        cli_credential = []
+        for item in cli_details:
             if item is None:
-                cliCredential.append(None)
+                cli_credential.append(None)
             else:
                 value = {
                     "username": item.get("username"),
                     "description": item.get("description"),
                     "id": item.get("id")
                 }
-                cliCredential.append(value)
-        return cliCredential
+                cli_credential.append(value)
+        return cli_credential
 
-    def get_snmpV2cRead_params(self, snmpV2cReadDetails):
+    def get_snmpV2cRead_params(self, snmpV2c_read_details):
```

Comment:
Can you please check the API docstring?
Also it should be 
snmp_v2c_read_details
get_snmp_v2c_read_params

Reply:
changed